source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+For the Mattermost mobile application, we’ll discuss  two options, a Device VPN or Per-app VPN.
+
+    **Note:** We suggest `following our recommended steps <https://docs.mattermost.com/mobile/mobile-appstore-install.html>`_ to secure your deployment.
+
+**Device VPN**
+    This is not as common, especially in the case of Bring Your Own Device (BYOD) scenarios. In this option, all internet traffic routes through the VPN specified in the profile. This could cause issue for personal applications. Of course even for devices provided by your business, this tends not to be the modern approach.
+
+**Per-app VPN**
+    In contrast, the more common approach is to use a per-app VPN. This provides a connection to the VPN when needed (on-demand). For example, when using a particular app.
+
+Regardless of the commonness of either option above, Mattermost provides support for both. Because Mattermost supports the AppConfig standard, both options are compatible with EMM providers.
+
+    **Note:** Will you be connecting via a corporate proxy server? If so, `review our FAQ <https://docs.mattermost.com/mobile/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`_ covering architecture, troubleshooting and best practices.
+
+----
+
+##########################################
+Choosing the Right Mobile Deployment Model
+##########################################
+
+At this point you should have read through the deployment primer. It provides a large amount of context for the principles and best practices that follow. This next section aims to help you choose one of two recommended deployment models.
+
+*****************************************
+Using the Public App Stores (Recommended)
+*****************************************
+
+**Desired Outcomes:**
+
+- Quick rollout of the Mattermost mobile application.
+- Allow users to install the application on their device.
+- Ensure a high level of security and controlled access.
+- Use existing, internal processes and tools.
+
+We recommend either of these options if you are…
+
+- Testing out the mobile applications
+- Deploying Team Edition servers using no push notifications, or push notifications from Mattermost’s `TPNS <https://docs.mattermost.com/overview/faq.html#tpns>`_ (Test Push Notification Service)
+- Deploying Enterprise Edition servers using push notifications from Mattermost’s `HPNS <https://docs.mattermost.com/mobile/mobile-hpns.html>`_ (Hosted Push Notification Service)
+
+The mobile applications provided by Mattermost work with our hosted version of the Mattermost Push Notification Service (MPNS). This represents the easier path. The Mattermost mobile applications can be deployed with or without an EMM provider. These options are explained in more detail below. 
+
+    **Note:** HPNS is compatible with EMM providers.
+
+**Option 1** - Public App Store Installation (Easiest)
```

Comment:
Again, lets get rid of `Public App Store`

Reply:
I’m all for this, but this is the exact language from our docs: “Mattermost mobile apps can be configured in your EMM solution using AppConfig with the apps on the *public app stores* (Google Play and Apple App Store), or as an “in-house app” you compile yourself.” https://docs.mattermost.com/mobile/mobile-appconfig.html 

Given this we stick with consistency (per other comments) and if an overall edit is needed we create and item and do that.

Reply:
This isn't repeating the mistake. I believe there should be a larger conversation around how we as an organization use these terms. Simply changing them here not only makes them inconsistent with the existing docs, but also how we talk about it internally.

Reply:
resolved in meeting